netbout-web/src/main/java/com/netbout/email/EmAlias.java

```diff
@@ -145,20 +140,31 @@ public void email(final String email, final String link)
                     )
                     .with(
                         new EnHTML(
-                            Joiner.on('\n').join(
+                            StringUtils.join(
```

Comment:
@amihaiemil any reason why you decided to use `StringUtils` instead of a `Joiner`?